service/user.ts

```diff
@@ -55,23 +64,11 @@ export type HostingPreferenceData = Omit<
  * Login user using password
  */
 export async function passwordLogin(username: string, password: string) {
-  const req = new AuthReq();
-  req.setUser(username);
-  req.setPassword(password);
-
-  const res = await client.auth.authenticate(req);
-  return res.toObject();
-}
-
-/**
- * Login user using a login token
- */
-export async function tokenLogin(loginToken: string) {
-  const req = new CompleteTokenLoginReq();
-  req.setLoginToken(loginToken);
-
-  const res = await client.auth.completeTokenLogin(req);
-  return res.toObject();
+  return http.post<LoginReq, LoginRes>(
```

Comment:
```suggestion
  return http.post(
```
Same comment about the generics as in the other file